File: RB101_Programming_Foundations/small_problems/Easy2/Run2/when_will_i_retire.rb
print "What is your age? "
age = gets.chomp.to_i

print "At what age would you like to retire? "
retire_age = gets.chomp.to_i

years_to_retire = retire_age - age

current_year = Time.new.year
retire_year = current_year + years_to_retire
puts "It's #{current_year}. You will retire in #{retire_year}"
puts "You have only #{years_to_retire} years of work to go!"

File: js210_fundamentals_of_JS_Programming/problem_sets/small_problems/Easy3/when_will_i_retire.js
let rls = require('readline-sync');

let age = Number(rls.question('What is your age? '));
let retireAge = Number(rls.question('At what age would you like to retire? '));

let currentYear = new Date().getFullYear();
let yearsTillRetire = retireAge - age;

console.log(`I'ts ${currentYear}. You will retire in ${currentYear + yearsTillRetire}.`);
console.log(`You have only ${yearsTillRetire} years of work to go!`);
